rust-toolchain.toml

```
@@ -1,6 +1,6 @@
 [toolchain]
 # We should use stable here once nitro-testnode is updated and the contracts fit
 # the size limit (issue <https://github.com/OpenZeppelin/rust-contracts-stylus/issues/129>).
-channel = "nightly-2024-09-05"
+channel = "stable"
```

```suggestion
channel = "1.83.0"
```
this is what Stylus SDK uses